crates/cow-amm/src/lib.rs

```diff
@@ -0,0 +1,38 @@
+mod cow_amm;
+pub mod cow_amm_constant_product_factory;
+mod event_updater;
+mod indexer;
+
+use ethcontract::Address;
+pub use {event_updater::EventUpdater, indexer::Indexer};
+
+pub trait CowAmm {
+    /// Address of the CoW AMM.
+    /// Can be used by the autopilot to build the list of accepted cow amms.
+    fn address(&self) -> &Address;
+
+    /// Returns the list of tokens traded by this pool.
+    /// Can be used by the autopilot to build the list of native token prices to
+    /// query.
+    fn traded_tokens(&self) -> &[Address];
+
+    // /// Computes a CoW protocol order and its required commitment interaction of
```

Comment:
This should either be deleted or checked in as it's supposed to be (but with a dummy implementation)